website/frontend/styles/ApiSection.scss

```diff
@@ -101,15 +97,11 @@
             }
         }
         
-        img {
-            display: none;
-        }
-
-        .api-section-mobile {
-            display: block;
+        .api-section-img {
             width: 100%;
             height: 100%;
-            object-fit: cover;
+            object-fit: contain;
```

Comment:
The mobile image should also be aligned to the center so that the overflow is hidden on both ends

Reply:
True. Though, the figma designs have the image trimmed in that way.

Reply:
Was this resolved @Codebmk?